schemas/context/media-timed-asset-reference.schema.json

```diff
@@ -0,0 +1,106 @@
+{
+  "meta:license": [
+    "Copyright 2018 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/media-timed-asset-reference",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Timed Media Primary Asset Reference",
+  "type": "object",
+  "meta:status": "experimental",
+  "meta:extensible": true,
+  "description": "Asset information about the main content that was played, but present on all ads and chapters that occur during the content's playback as well.",
+  "definitions": {
+    "media-asset-reference": {
+      "properties": {
+        "@id": {
+          "title": "Content ID",
+          "type": "string",
+          "format": "uri",
+          "description": "Content ID of the content, which can be used to tie back to other industry / CMS IDs."
+        },
+        "xdm:name": {
+          "title": "Media Name",
+          "type": "string",
+          "description": "The friendly (human-readable) name of the timed media asset."
+        },
+        "xdm:duration": {
+          "title": "Media Length/Runtime",
+          "type": "integer",
+          "description": "Length of primary media asset in seconds."
+        },
+        "xdm:showName": {
+          "title": "Show Name",
+          "type": "string",
+          "description": "Program/Series Name."
+        },
+        "xdm:assetID": {
```

Comment:
@cosminci @gabebarcelos-adobe  do we want to move to use s/content/asset/ in the schema names and identity here?

Reply:
For Media Analytics, the assetID is reported through custom metadata, and will not be available in the majority of timed media experience events. 
We could update the properties to be more consistent with the existing schemas.

Reply:
@cdegroot-adobe @kstreeter - I'm not sure I follow. Are you suggesting adding an existing schema like [Video](https://github.com/adobe/xdm/blob/b0022331abfe1281b585da4b8f81db9e94d8a4b6/schemas/assets/video.schema.json) or [Image](https://github.com/adobe/xdm/blob/master/schemas/assets/image.schema.json) as a sub-schema to these and moving the appropriate attributes under those? Or are we discussing more of nomenclature of the existing schema?

Reply:
Hi @gabebarcelos-adobe, I am not actually suggesting we include existing schemas directly, since we don't actually have most of the properties of an asset item in these schema.  So it is just nomenclature.  The use of the term "asset" in these schemas is describing a very different concept than it does in the XDM asset schemas.  Also, the use of the term isn't completely consistent within the new schemas themselves, with multiple concepts all being called "asset".

Reply:
I'm going to re-open it.  I am OK calling it asset, but it *can't* be assetID, as that is already used in the asset schema.  It needs a different/unique name unless you are using it the same way - then it's OK.

Reply:
One possibility is to use Iptc4xmpExt:Identifier

Reply:
The right solution seemed to be to move to the IPTC Series structure. This gave me a way to resolve showName and assetID in a consistent way. 
      "iptc4xmpExt:Series": {
        "iptc4xmpExt:Name": "nba_highlights",
        "iptc4xmpExt:Identifier": "http://espn.com/series-identifiers/2613953",
      },